scripts/setup.sh

```diff
@@ -179,9 +179,9 @@ if [ ! -d "$NODE_PATH" ]; then
 fi
 
 # Adjust path to support the default Chrome locations for Unix, Windows and Mac OS.
-if [ "$TRAVIS" = true ]; then
+if [ "$TRAVIS" == true ]; then
```

Comment:
Should we quote this as well?

Reply:
I only added in the double quotes per your request. I do not think we should constantly make changes to prevent instability.

We can make new code follow a certain style/guideline. To constantly go in and make small changes to existing code just for the sake of styling is not advisable imo.

Reply:
Sounds good, thanks!